HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # History
 
+## v4.0.0 2020 June 19
+
+-   Converted from JavaScript to TypeScript
+-   If you are using CommonJS, you must now do require('trim-indentation').default
```

Comment:
`require('valid-directory').default` - remember the backticks, remember the change the name to package